src/igvfd/schemas/measurement_set.json

```diff
@@ -186,7 +186,7 @@
                 "scATAC-seq": "Single-cell ATAC-seq (scATAC-seq) profiles chromatin accessibility at single-cell resolution.",
                 "scACCESS-ATAC": "Single-cell Accessible Chromatin by Cytosine Editing Site Sequencing with ATAC-seq (scACCESS-ATAC) is used to profile chromatin accessibility and transcription factor binding at high resolution in single cells.",
                 "snATAC-seq": "Single-nucleus ATAC-seq (snATAC-seq) profiles chromatin accessibility in individual nuclei.",
-                "10x multiome with scMito-seq": "Combines 10x multiome with single-cell mitochondrial sequencing (scMito-seq) profiles mitochondrial DNA (mtDNA) variants and transcriptional activity in individual cells.",
+                "mtscMultiome": "mitochondrial single-cell Multiome (mtscMultiome) combines 10x multiome with single-cell mitochondrial sequencing (scMito-seq) profiles mitochondrial DNA (mtDNA), chromatin accessibility, variants and transcriptional activity in individual cells.",
```

Comment:
In the Jira description there is some mention of "add a comma before profiles", which I am not really sure makes grammatical sense. But the current sentence also seems wrong to me. 

Maybe it should be "mitochondrial single-cell Multiome (mtscMultiome) combines 10x multiome with single-cell mitochondrial sequencing (scMito-seq) **to profile** mitochondrial DNA (mtDNA), chromatin accessibility, variants and transcriptional activity in individual cells." Does that work?

Reply:
yeah! thanks